packages/store/src/data-provider/system-migration.ts

```diff
@@ -0,0 +1,43 @@
+import { DataProvider, DataProviderConstructor } from '.';
+import { DataAccess } from '../data-access';
+import { FilesystemStore } from '../data-access/filesystem';
+import { SystemMigrationStatus, SystemMigrationStatusKey } from '../entities/system-migration-status';
+import { Logger } from '@sap-ux/logger';
+import { Entities } from './constants';
+
+export const SystemMigrationStatusDataProvider: DataProviderConstructor<
```

Comment:
Same comment, jsdoc missing throughout.

Reply:
Same comment as this: https://github.com/SAP/open-ux-tools/pull/237#discussion_r791175087

Reply:
Added a comment in the interface in https://github.com/SAP/open-ux-tools/pull/237/commits/e37e1de9ea46e33d3a9e87c4febfd4187ff5186a